### ISSUE TITLE
Add a button to expand the document preview full-screen

### ISSUE THREAD

**Issue by chasenlehara** [CCALI/A2JViewer#182]
This might be something provided by lightbox and not straightforward to add—investigation will be required.

**Comment by janebitovi** [comment]
incoming change preview:
![fullscreen](https://user-images.githubusercontent.com/92131113/147587328-c612e09a-bdbd-43a2-ac12-5e50397eccf7.gif)

content on the inside has a max-width of 7.5 inches which should be a close match to actual documents printed on 8.5 inch paper with 0.5 inch printing margin
